src/templates/haml/home2/index.html.haml

```diff
@@ -0,0 +1,120 @@
+- stash(
```

Comment:
I don't think you need a home2. Home2 is just for a/b testing. This project isn't about a/b testing.